lib/WindowsCompat.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  pathToCanonical(path: string): string {
+    return path.replace(/(^[A-Z]:\\|\\)/g, '/');
```

Comment:
Since backslashes are allowed in *nix filenames, I think we should guard here against replacing so that it only happens on windows. Would that make sense?